src/analysis/normed_space/operator_norm.lean

```diff
@@ -1261,6 +1261,74 @@ begin
   exact ⟨Glin, tendsto_of_tendsto_pointwise_of_cauchy_seq (tendsto_pi_nhds.2 hG) hf⟩
 end
 
+/-- Let `s` be a bounded set in the space of continuous (semi)linear maps `E →SL[σ] F` taking values
+in a proper space. Then `s` interpreted as a set in the space of maps `E → F` with topology of
+pointwise convergence is precompact: its closure is a compact set. -/
+lemma is_compact_closure_image_coe_bounded [proper_space F] {s : set (E' →SL[σ₁₂] F)}
+  (hb : bounded s) :
+  is_compact (closure ((coe_fn : (E' →SL[σ₁₂] F) → E' → F) '' s)) :=
+have ∀ x, is_compact (closure (apply' F σ₁₂ x '' s)),
+  from λ x, ((apply' F σ₁₂ x).lipschitz.bounded_image hb).is_compact_closure,
+compact_closure_of_subset_compact (is_compact_pi_infinite this)
+  (image_subset_iff.2 $ λ g hg x, subset_closure $ mem_image_of_mem _ hg)
+
+/-- Let `s` be a bounded set in the space of continuous (semi)linear maps `E →SL[σ] F` taking values
+in a proper space. If `s` interpreted as a set in the space of maps `E → F` with topology of
+pointwise convergence is closed, then it is compact. -/
+lemma is_compact_image_coe_bounded_of_closed_image [proper_space F] {s : set (E' →SL[σ₁₂] F)}
+  (hb : bounded s) (hc : is_closed ((λ (f : E' →SL[σ₁₂] F) x, f x) '' s)) :
+  is_compact ((coe_fn : (E' →SL[σ₁₂] F) → E' → F) '' s) :=
+hc.closure_eq ▸ is_compact_closure_image_coe_bounded hb
+
+/-- If a set `s : set (E →SL[σ] F)` is bounded and is closed in the weak-* topology, then its image
```

Comment:
It seems more useful to reformulate the Lean statement in human language than copy it. What do you think of this?
```suggestion
/-- If a set `s` of semilinear functions is bounded and is closed in the weak-* topology, then its image
```